scripts/lib/CIME/case.py

```diff
@@ -1006,14 +1008,11 @@ def apply_user_mods(self, user_mods_dir=None):
         or they can be in the compset definition, or both.
         """
 
-        if self._user_mods is None:
-            compset_user_mods_resolved = None
-        else:
-            compset_user_mods_resolved = self.get_resolved_value(self._user_mods)
+        component_user_mods = self.get_value("%s_USER_MODS"%(self._principle_component))
```

Comment:
I didn't understand why it had that and don't think that it needed it.,  the default returned from get_value is resolved=True